client/components/thank-you-v2/header/style.scss

```diff
@@ -36,6 +36,11 @@
 	text-align: left;
 	letter-spacing: -0.1px;
 
+	.button-plain {
+		color: var(--studio-blue-50);
+		cursor: pointer;
+	}
```

Comment:
By using `<Button>` instead of a simple `<a>`, we need to add css to ensures the button renders like an anchor tag.